demo.svelte

```diff
@@ -422,7 +434,13 @@
             </div>
             <div id="result-table" class="card">
                 <lens-result-table pageSizeSwitcher={true}></lens-result-table>
-                <lens-negotiate-button></lens-negotiate-button>
```

Comment:
I would use the lens-negotiate-button, then we test it and show how to use it. And we don't the styling.